app/models/spouse.rb

```diff
@@ -1,2 +1,3 @@
 class Spouse < Person
+  belongs_to :family
 end
```

Comment:
A spouse will be treated like an attendee, right? If that is the case, would it not be simpler to make the spouse class a descendant of the attendee class?